bin/toolbox/presubmit/package_coverage_check.py

```diff
@@ -0,0 +1,15 @@
+from istio_toolbox.pkgCheck import pkg_cc_check
```

Comment:
Yes, you are right, it's definitely repo-independent. However, I just can't find a direct way to call a program from another repo. That's why I put this small part of code as a entry such that I can call the really function in toolbox repo.